channels/af.m3u

```diff
@@ -11,8 +11,6 @@ https://d2g7v53450s2i2.cloudfront.net/ATNUS/streamdelay/playlist.m3u8
 https://d10rltuy0iweup.cloudfront.net/ATNNEWS/myStream/playlist.m3u8
 #EXTINF:-1 tvg-id="BaharTV.af" tvg-country="AF" tvg-language="" tvg-logo="" group-title="",Bahar TV (720p)
 https://59d39900ebfb8.streamlock.net/bahartv/bahartv/playlist.m3u8
-#EXTINF:-1 tvg-id="BaryaTV.af" tvg-country="AF" tvg-language="" tvg-logo="https://i.imgur.com/yf5cI8r.jpg" group-title="",Barya TV (720p) [Offline]
-http://51.210.199.56/hls/stream.m3u8
```

Comment:
It's better to leave it on the playlist, too

Reply:
Ok